<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabShape" >
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Access Position</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="AccessPosition" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labell" >
           <property name="text" >
            <string>Geotechnical Field</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="geotechProp" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Density</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="densityInput" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Block Size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QDoubleSpinBox" name="BlockSizeX" />
         </item>
         <item row="0" column="2" >
          <widget class="QDoubleSpinBox" name="BlockSizeY" />
         </item>
         <item row="0" column="3" >
          <widget class="QDoubleSpinBox" name="BlockSizeZ" />
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <widget class="QCheckBox" name="RegularBlock" >
         <property name="text" >
          <string>Regular Blocks</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>341</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Python</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; color:#0000ff;">Python Path&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="pythonPath" />
         </item>
         <item row="0" column="2" >
          <widget class="QToolButton" name="browsePythonPath" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <layout class="QGridLayout" >
           <item row="0" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>447</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <layout class="QGridLayout" >
               <item row="0" column="0" >
                <widget class="QPushButton" name="editScriptButton" >
                 <property name="text" >
                  <string>Edit Script</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" >
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>17</width>
                   <height>148</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1" >
              <widget class="QListWidget" name="filesList" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" color:#0000ff;">Preview&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QTextEdit" name="previewTextEdit" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags" >
              <set>Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5" >
      <attribute name="title" >
       <string>Functions</string>
      </attribute>
      <widget class="QTreeWidget" name="functionsTree" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>421</width>
         <height>451</height>
        </rect>
       </property>
       <column>
        <property name="text" >
         <string>Fitness Functions</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDockWidget" name="dockWidget" >
     <property name="floating" >
      <bool>true</bool>
     </property>
     <property name="features" >
      <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
     </property>
     <widget class="QWidget" name="dockWidgetContents" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTextEdit" name="textEdit" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>148</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QPushButton" name="saveAs" >
           <property name="text" >
            <string>Save As</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="save" >
           <property name="text" >
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <widget class="QPushButton" name="discard" >
           <property name="text" >
            <string>Discard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
